packages/card/package.json

```diff
@@ -69,6 +69,7 @@
         "@spectrum-web-components/checkbox": "1.6.0",
         "@spectrum-web-components/divider": "1.6.0",
         "@spectrum-web-components/icons-workflow": "1.6.0",
+        "@spectrum-web-components/popover": "1.6.0",
```

Comment:
Does every card require a popover? It almost seems like this should be an optional dependency? When does a card use a popover?

Reply:
It should be a regular dependency, not an optional one. The component needs to be available when the `toggles` feature is used. As per my understanding optional dependencies are typically for features which can be completely replaced or deleted. We have never shipped any components with optionalDependencies